recipes/jwt-cpp/all/patches/0004-fix-picojson-header-location-for-conan.patch

```diff
@@ -0,0 +1,6 @@
+diff --git a/include/jwt-cpp/jwt.h b/include/jwt-cpp/jwt.h
+--- a/include/jwt-cpp/jwt.h
++++ b/include/jwt-cpp/jwt.h
+@@ -8,1 +8,1 @@
+-#include "picojson/picojson.h"
++#include "picojson.h"
```

Comment:
Is picojson.h in a wrong path indeed, or Conan packaged it wrongly?

Reply:
No, it's the right path.

Unfortunately, examples don't answer your question:
https://github.com/kazuho/picojson/blob/master/examples/github-issues.cc#L29
`#include "../picojson.h"`

But Makefile does:
It installs picojson.h right in includedir (`/usr/local/include` for example)
https://github.com/kazuho/picojson/blob/master/Makefile#L21

Reply:
It's because JWT-CPP has a vendored  version at that path. The conan one is correct.